.vsts-ci/linux.yml

```diff
@@ -18,6 +18,8 @@ phases:
     matrix:
       Linux ARM Native:
         buildName: ubuntu.16.04
+      Linux ARM64 Native:
+        buildName: ubuntu.16.04
```

Comment:
Use the name defined in build.json instead of `ubuntu.16.04`. I believe this will be building arm32 again.